cloud/docker/docker.py

```diff
@@ -682,6 +682,10 @@ def get_deployed_containers(self):
             name = '/' + name
         deployed = []
 
```

Comment:
Need to set all_valid_tags to an empty set()/frozenset() here just in case we don't detect any images that match.